Makefile.am

```diff
@@ -726,7 +726,8 @@ vmdisplay_wayland_LDADD = $(COMPOSITOR_LIBS) $(GLIB_LIBS) $(EGL_LIBS) $(SIMPLE_C
 	libshared.la libvmdisplay.la -lm -ldrm_intel \
 	$(WAYLAND_COMPOSITOR_EGL_LIBS)		\
 	$(WAYLAND_COMPOSITOR_LIBS)		\
-	$(SIMPLE_EGL_CLIENT_LIBS)
+	$(SIMPLE_EGL_CLIENT_LIBS)               \
+	libias-@LIBWESTON_MAJOR@.la
```

Comment:
@YinThong  please elaborate on this change.  It looks to me as a hack.   Can it be linked dynamically?

Reply:
@chivakker yes is linked dynamically. When i build with --enable-vmdisplay it having below compiler issue.

/usr/bin/ld: ./.libs/libvmdisplay.a(libvmdisplay_la-vmdisplay.o): in function zwp_linux_dmabuf_v1_create_params': /builddir/build/BUILD/ias-4.0.9/./protocol/linux-dmabuf-unstable-v1-client-protocol.h:344: undefined reference to zwp_linux_buffer_params_v1_interface'
/usr/bin/ld: clients/vmdisplay/vmdisplay_wayland-vmdisplay-wayland.o: in function wl_registry_bind': /usr/include/wayland-client-protocol.h:1091: undefined reference to zwp_linux_dmabuf_v1_interface'

As the function is allocated inside libweston. Base on the Makefile.am design all enable is compiler with own dependency linking.

Additional:-
this is my autogen.sh parameter:-
./autogen.sh --disable-setuid-install --enable-ias-shell --disable-xkbcommon --enable-simple-clients --enable-clients --disable-wayland-compositor --disable-libunwind --disable-xwayland --disable-xwayland-test --disable-x11-compositor --disable-rpi-compositor --enable-ivi-plugin-manager --enable-layer-manager-control --enable-tracing --enable-shadergen --enable-demo-clients-install --enable-vm --enable-vmdisplay